src/utils/utils.ts

```diff
@@ -154,8 +159,21 @@ export function patch(target, methodName, mixinMethod) {
 
     Object.defineProperty(target, methodName, newDefinition)
 }
+interface Defintion {
```

Comment:
`Definition`. But this type isn't needed at all I think, there is already the built in `PropertyDescriptor`

Reply:
Yes, you're right.